docs/superpowers/plans/2026-08-17-worker-crash-task-recovery.md

```
@@ -0,0 +1,1068 @@
+
+---
+
+### Task 6: Documentation
+
+**Files:**
+- Modify: `AGENTS.md` (`CLAUDE.md`/`GEMINI.md` are symlinks to it — do not touch them): sections "Job/Task Processing Model" (line ~66), "Task Actions" table Kill row (line ~108), "Docker Services" (line ~125), "Environment Variables" (line ~135)
+- Modify: `KNOWLEDGE.md` (add a "Procrastinate" section under `## Django`, or next to `## Docker`)
+
+- [ ] **Step 1: AGENTS.md — processing model**
+
+Replace in "Job/Task Processing Model": `Background workers (Procrastinate) poll and process tasks from two queues: `default` and `dicom`` with `three queues: `default`, `dicom` and `mass_transfer``.
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)